connectors/migrations/20240216_make_notion_cache_tables_unlogged.ts

```diff
@@ -0,0 +1,20 @@
+import { sequelizeConnection } from "@connectors/resources/storage";
+
+async function main() {
+  await sequelizeConnection.query(`
+        ALTER TABLE notion_connector_page_cache_entries SET UNLOGGED;
```

Comment:
Just to be sure that this is what we want, `UNLOGGED` tables can only be accessed on the primary.

Reply:
Yep